grpc/recovery_test.go

```diff
@@ -1,28 +1,43 @@
-package server_test
+package grpc
 
 import (
+	"bytes"
 	"context"
+	"os"
+	"strings"
+	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
-	. "github.com/contiamo/goserver/grpc"
 	"github.com/contiamo/goserver/grpc/test"
 )
 
 var _ = Describe("Recovery", func() {
 	It("should recover from panic when recovery option is set", func() {
+		buf := &bytes.Buffer{}
+		logrus.SetOutput(buf)
+		defer func() {
+			logrus.SetOutput(os.Stdout)
+		}()
+
 		srv, err := createServerWithOptions([]Option{WithRecovery()})
 		Expect(err).NotTo(HaveOccurred())
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
+
 		go ListenAndServe(ctx, ":3005", srv)
+		time.Sleep(time.Second)
```

Comment:
Add a comment here so that this is not deleted in the future

Reply:
Added the comment and `time.Sleep` in other places where it's necessary. Also, decreased the time to 100 milliseconds which should be enough.